one-ui/ui/src/components/entities/mapping/mapping-card.test.tsx

```diff
@@ -12,6 +12,7 @@ describe('Mapping cards view component', () => {
       updateMappingArtifact
       canReadOnly
       canReadWrite
-      entityModel/>);
+      entityModel
+      mapNames/>);
```

Comment:
does this test pass when running locally? The props "data" is not right for the mapping card. Try using RTL to render like so and verify if you get back mapping card HTML or an empty div
```
import { render } from "@testing-library/react"; 

describe.... 

test('it renders mapping card div correctly' , () => {
const { debug } = render(<MappingCard data />);
debug();
};
```

Reply:
@bsrikan - Yes, it already passes as is when running locally.